99-make_reports.py

```diff
@@ -124,7 +124,7 @@ def run_report(subject, session=None):
         if mne.viz.get_3d_backend() is not None:
             fig = mne.viz.plot_alignment(evoked.info, fname_trans,
                                          subject=subject,
-                                         subjects_dir=config.subjects_dir,
+                                         subjects_dir=subjects_dir,
```

Comment:
We're now correctly using the `subjects_dir` variable, which is generated further up via `config.get_fs_subjects_dir()`.